source/unified-test-format/tests/valid-fail/entity-find-cursor.yml

```diff
@@ -0,0 +1,41 @@
+description: entity-find-cursor
+
+schemaVersion: '1.3'
+
+createEntities:
+  - client:
+      id: &client0 client0
+      observeEvents:
```

Comment:
This seems irrelevant since there are no event assertions below.

Reply:
Thanks for catching that. Removed the `observeEvents` array.